cpp/tests/test_odesecir.cpp

```diff
@@ -673,7 +620,7 @@ TEST(TestSecir, testModelConstraints)
     // }
 
     // temporary test for random variables
-    set_params_distributions_normal(model, t0, tmax, 0.2);
+    set_params_distributions_normal(model, t0, tmax, 0.1);
```

Comment:
for incubation time = 5.2, the serial interval must be between 3.1 and 4.2, so that it does not throw an error in the check constraints. If we now sample the parameters any deviation, 
we have with very high probability not allowed values in the simulation. Since 4.2 is the maximum allowed value.
and if the SI is outside this interval, then the test takes almost infinite time and then aborts as failed.
so i reduced the deviation and chose the value for the serial interval exactly in the middle of the allowed interval in the tests.
apply_constraints had always corrected such values before.